demo/demo.js

```diff
@@ -126,60 +125,138 @@ ethSignMsg.addEventListener("click", async () => {
         getKeypathFromString("m/49'/0'/0'/0/0"),
-        api.firmware.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
+        constants.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
     );
 });
 
-document.getElementById("btcSignSimple").addEventListener("click", async () => {
-    const bip44Account = 0 + HARDENED;
-    const version = 1;
-    const locktime = 0;
+// Mock sample single-sig BTC transaction
+function makeExampleTx(keypathAccount) {
     const inputs = [
         {
             "prevOutHash": new Uint8Array(32).fill(49), // arbitrary constant
             "prevOutIndex": 1,
-            "prevOutValue": 1e8 * 0.60005,
+            "prevOutValue": "60005000", // satoshis as a decimal string
             "sequence": 0xFFFFFFFF,
-            "keypath": [84 + HARDENED, 0 + HARDENED, bip44Account, 0, 0],
+            "keypath": keypathAccount.concat([0, 0]),
         },
         {
             "prevOutHash": new Uint8Array(32).fill(49), // arbitrary constant
             "prevOutIndex": 1,
-            "prevOutValue": 1e8 * 0.60005,
+            "prevOutValue": "60005000", // satoshis as a decimal string
             "sequence": 0xFFFFFFFF,
-            "keypath": [84 + HARDENED, 0 + HARDENED, bip44Account, 0, 1],
+            "keypath": keypathAccount.concat([0, 1]),
         }
     ];
     const outputs = [
         {
             "ours": true, // change
-            "keypath": [84 + HARDENED, 0 + HARDENED, bip44Account, 1, 0],
-            "value": 1e8 * 1,
+            "keypath": keypathAccount.concat([1, 0]),
+            "value": "100000000", // satoshis as a decimal string
         },
         {
             "ours": false,
-            "type": api.firmware.messages.BTCOutputType.P2WSH,
+            "type": constants.messages.BTCOutputType.P2WSH,
             "hash": new Uint8Array(32).fill(49), // arbitrary constant
-            "value": 1e8 * 0.2,
+            "value": "20000000", // satoshis as a decimal string,
         },
     ];
+    const version = 1;
+    const locktime = 0;
+    return { inputs, outputs, version, locktime };
+}
+
+// Sign single-sig BTC transaction
+document.getElementById("btcSignSimple").addEventListener("click", async () => {
+    const keypathAccount = getKeypathFromString("m/84'/0'/0'");
+    const { inputs, outputs, version, locktime } = makeExampleTx(keypathAccount);
+    try {
+        const signatures = await device.api.btcSignSimple(
+            constants.messages.BTCCoin.BTC,
+            constants.messages.BTCScriptConfig_SimpleType.P2WPKH,
+            keypathAccount,
+            inputs,
+            outputs,
+            version,
+            locktime,
+        );
+        console.log("Signatures: ", signatures);
+    } catch(err) {
+        if (isErrorAbort(err)) {
+            alert("aborted by user");
+        } else {
+            alert(err.Message);
+        }
+    }
+});
+
+// Mock sample multi-sig BTC transaction
+async function makeExampleMultisigAccount(keypathAccount) {
```

Comment:
Genrerally I'd prefer to use ES6 arrow function declarations for consistency:
`const makeExampleMultisigAccount = async (keypathAccount) => {...}`

Same goes for all the `function` declarations